openedx/core/djangoapps/programs/tests/test_models.py

```diff
@@ -8,10 +8,9 @@
 from openedx.core.djangoapps.programs.models import ProgramsApiConfig
 
 
-@patch('config_models.models.cache.get', return_value=None)  # during tests, make every cache get a miss.
-class ProgramsApiConfigTest(TestCase):
+class ProgramsApiBasicTestConfig(TestCase):
```

Comment:
I think this should be a mixin (`ProgramsApiConfigMixin`) instead of a base class, and reside in tests/mixins.py